src/server/pfs/fuse/server_test.go

```diff
@@ -1031,3 +1014,57 @@ func TestProjects(t *testing.T) {
 		require.Equal(t, projectData[2].Project.Name, emptyProjectName)
 	})
 }
+
+func TestMountingCommit(t *testing.T) {
```

Comment:
Should there also be a test that trying to mount a rw commit fails?

Reply:
Mounting a `rw` commit won't fail. It'll just mount the latest on the branch that's used, whether it's the one the user specifies or `master` (default if no branch specified). I'll add a test to validate that the latest commit on a branch is the one mounted in `rw` mode, rather than a specified earlier commit.